src/FootPrintsInterpolator.cpp

```diff
@@ -570,29 +616,113 @@ bool FeetInterpolator::computeLocalZMP(const std::vector<StepPhase> &stepPhase,
     return true;
 }
 
-void FeetInterpolator::computeGlobalZMP()
+iDynTree::Position FeetInterpolator::pos3D(const iDynTree::Vector2 &xy)
+{
+    iDynTree::Position output;
+    output.zero();
+    output(0) = xy(0);
+    output(1) = xy(1);
+    return output;
```

Comment:
I personal prefer the compact version `return iDynTree::Position(xy(0), xy(1), 0.0);`, but I would understand the reason for preferring the long version.